backoffice/tests/UnitTests/Services/MigrationServiceTests.cs

```diff
@@ -0,0 +1,78 @@
+using AutoMapper;
+using Droits.Exceptions;
+using Droits.Models;
+using Droits.Models.Entities;
+using Droits.Services;
+using Microsoft.Extensions.Logging;
+
+namespace Droits.Tests.UnitTests.Services
+{
+    public class MigrationServiceUnitTests
+    {
+        private readonly MigrationService _service;
+        private readonly Mock<IDroitService> _mockDroitService;
+        private readonly Mock<ILogger<MigrationService>> _mockLogger;
+
+        public MigrationServiceUnitTests()
+        {
+            _mockDroitService = new Mock<IDroitService>();
+            var mockFileService = new Mock<IDroitFileService>();
+            var mockWreckMaterialService = new Mock<IWreckMaterialService>();
+            var mockSalvorService = new Mock<ISalvorService>();
+            var mockImageService = new Mock<IImageService>();
+            var mockWreckService = new Mock<IWreckService>();
+            var mockNoteService = new Mock<INoteService>();
+            var mockMapper = new Mock<IMapper>();
+            _mockLogger = new Mock<ILogger<MigrationService>>();
+
+            _service = new MigrationService(_mockLogger.Object,_mockDroitService.Object,
+                mockWreckMaterialService.Object, mockSalvorService.Object, mockImageService.Object,
+                mockFileService.Object,  mockWreckService.Object,mockNoteService.Object, mockMapper.Object);
+        }
+
+
+        [Fact]
+        public async Task HandleTriageCsv_UpdatesGivenDroits()
+        {
+            // Assemble
+            var records = new List<TriageRowDto>()
+            {
+                new()
+                {
```

Comment:
try some ranges outside of expected, also references with letters in

Reply:
references with letters is kinda covered with the droit ref exception test as it only matches the string together, we could add regex pattern checking to it but 🤷